03-objects-and-composition/exercise/03. Car Factory

```diff
@@ -0,0 +1,29 @@
+function carFactory(car) {
```

Comment:
Your file currently is not .js file. Please fix it.

Reply:
Fixed, thank you!